tooling-builders/src/main/kotlin/org/gradle/kotlin/dsl/tooling/builders/KotlinLibSources.kt

```diff
@@ -58,20 +58,18 @@ fun sourcePathFor(scriptHandlers: List<ScriptHandler>): ClassPath {
 
 
 private
-fun containsBuiltinKotlinModules(resolvedDependencies: HashSet<ModuleVersionIdentifier>) =
-    resolvedDependencies.containsAll(
-        builtinKotlinModules.map(::kotlinModuleVersionIdentifier))
-
-
-private
-fun classpathDependenciesOf(buildscript: ScriptHandler): List<ModuleVersionIdentifier> =
-    DefaultIdeDependencyResolver()
-        .getIdeRepoFileDependencies(buildscript.configurations[CLASSPATH_CONFIGURATION])
-        .map { it.id }
+fun containsBuiltinKotlinModules(resolvedDependencies: HashSet<ComponentIdentifier>) =
```

Comment:
Why `HashSet` instead of just a `Set`? is there some specific behaviour of a `HashSet` you need??

Reply:
It's just an artifact of the refactoring.